cli/app/commands/install/run.py

```diff
@@ -120,16 +163,59 @@ def run(self):
                     except Exception as e:
                         progress.update(self.main_task, description=f"Failed at {step_name}")
                         raise
-                
+
                 progress.update(self.main_task, completed=True, description="Installation completed")
-            
+
             self._show_success_message()
-            
+
         except Exception as e:
             self._handle_installation_error(e)
             self.logger.error(f"{installation_failed}: {str(e)}")
             raise typer.Exit(1)
 
+    def _cleanup_docker(self):
```

Comment:
let's move this function to the base of service command? because similar logics related to docker exists there, and if at all this is required over there we can reuse this function, what do you think?